build.sbt

```diff
@@ -27,7 +27,7 @@ lazy val localConsignmentExport = (project in file("consignment-export"))
       "com.typesafe.akka" %% "akka-stream" % "2.6.3",
       "de.heikoseeberger" %% "akka-http-circe" % "1.32.0",
       "uk.gov.nationalarchives" %% "tdr-graphql-client" % "0.0.15",
-      "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.160",
+      "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.187",
```

Comment:
The versions for different libraries are set in multiple places. Is it possible to move any duplicated library versions into variables and use these? For example:
```
...
val genratedGraphQlVersion = "0.0.187"
...
"uk.gov.nationalarchives" %% "tdr-generated-graphql" % generatedGraphQlVersion,
...

```
It'll make sure all the different projects are running on the same versions of libraries, where those libraries are used by more than one project.